src/app/app.component.scss

```diff
@@ -2,6 +2,7 @@
 
 ::ng-deep .pxb-blue .mat-drawer-container {
     background-color: map-get($pxb-white, 50) !important;
+    min-height: 100vh !important;
 }
```

Comment:
I would prefer to handle this by setting a rule for:
```
html, body { height: 100%; }
```
(the drawer layout is already set to fill 100% of the height of its parent, so the 100vh is kind of limiting and overkill)

and you should be able to set
```
.pxb-drawer-layout-nav-content { bakground: white; }
```
to do the background, assuming your ViewEncapsulation is configured correctly.

Side note: I don't think our drawer layout should be setting a background color at all...it should be inheriting from whatever is set on the body by default.